newspaper/static/layout.css

```diff
@@ -62,32 +60,25 @@ a {
     margin-right: 6rem;
     background-color: #000000;
     color: white;
-    padding-right: 1rem;
-    padding-left: 1rem;
-    padding-top: 0.5rem;
-    padding-bottom: 0.5rem;
+    padding: 0.5rem 1rem ;
     -webkit-box-shadow: -15px 26px 64px -19px rgba(28,28,28,0.97);
     -moz-box-shadow: -15px 26px 64px -19px rgba(28,28,28,0.97);
     box-shadow: -15px 26px 64px -19px rgba(28,28,28,0.97);
 }
 
 .subscribe-navbtn:hover {
-    background-color: #272727;
+    background-color: var(--darkbrown);
     color: white;
 }
 
 
 .footer {
-    position: absolute;
-    background-color: #272727;
-    width: 100vw;
+    background-color: var(--darkbrown);
     height: 10vh;
-    bottom: 0;
```

Comment:
This will move the footer to an incorrect position, let this be there and also the `:root` colours are not defined. Check this out: https://www.w3schools.com/css/css3_variables.asp (Define all the variables inside a root file).